src/analyzer/wlanalyzer.cpp

```diff
@@ -53,9 +53,9 @@ void MainWindow::createMenus()
 
 void MainWindow::connectSlot()
 {
-	IpDialog ipDialog;
+	IpDialog* ip = new IpDialog();
 
-	ipDialog.exec();
+	ip->show();
```

Comment:
This dialog is a modal dialog and it's best to invoke exec() on it and not show(). We don't want the user access to other windows while typing in the IP address to connect to in a modal dialog.